Fix command in HOW_TO_USE.md

zswen opened the pull request:
Fixes #34 
-
Updated command in HOW_TO_USE.md.

arshadkazmi42 commented on HOW_TO_USE.md:
````diff
@@ -37,15 +37,15 @@ Dont't know what these stand for? No problem, following are the steps. 
 
       ```
-      python first_timers/run.py --creds-path credentials.json --db-path data/db.json --only-save
+      python first_timers/run.py --creds-path credentials.json --db-path data/db.json --only-save --create
````
We do not want to update every command with create flag.
Remove `--create` from this

arshadkazmi42 commented on HOW_TO_USE.md:
````diff
@@ -37,15 +37,15 @@ Dont't know what these stand for? No problem, following are the steps. 
 
         ``` 
-        python first_timers/run.py --creds-path credentials.json --db-path data/db.json
+        python first_timers/run.py --creds-path credentials.json --db-path data/db.json --create
````
Remove `--create`

arshadkazmi42 commented on HOW_TO_USE.md:
```diff
@@ -37,15 +37,15 @@ Dont't know what these stand for? No problem, following are the steps. 
     - To only fetch the issues and store without tweeting
```
Add before this line
```
- To create `db.json` file, use the below command (This command should be executed before all other commands)
`python first_timers/run.py --creds-path credentials.json --db-path data/db.json --only-save --create`
```

zswen commented:
Working on it!

zswen commented:
Hello @arshadkazmi42 . I have removed `--create flag` and added the db.json command to HOW_TO_USE.md. Thank you very much for your review. Please let me know if I did everything correctly.

zswen commented:
@arshadkazmi42 Thank you very much for approving my first PR. Thank for all your reviews. This is a big step for me. I will keep contribute.

arshadkazmi42 commented:
Congrats on landing your first PR and thanks for finding this issue
